### PR TITLE
Add content to discussion.md file.

### PR DESCRIPTION
**Pull request by douglatornell**

### CODE REVIEW

File: discussion.md [hunk 1]
```diff
@@ -3,4 +3,131 @@ layout: page
...
+Note that since this material isn't essential for basic Mercurial usage,
+it won't be covered by the instructor.
+
+
+## More Advanced Mercurial Configuration
+
+In the [Setting Up](01-backup.html#setting-up) section we edited a Mercurial configuration file in our home directory called `$USERPROFILE/Mercurial.ini` or `~/.hgrc`.
+You can quickly open that file for editing with the command `hg config --edit`.
+
+If you want to know more about the many configuration options available you can use `hg help config` or visit [this page](http://www.selenic.com/mercurial/hgrc.5.html).
```
Comment (kynan) [inline discussion.md]: this page -> the mercurial documentation

File: discussion.md [hunk 2]
```diff
@@ -3,4 +3,131 @@ layout: page
...
+
+On configuration option that can be useful is adding aliases.
```
Comment (kynan) [inline discussion.md]: One

File: discussion.md [hunk 3]
```diff
@@ -3,4 +3,131 @@ layout: page
...
+~~~ {.output}
+Cold and dry, but everything is my favorite color
+~~~
+
+Save the document into the `planets-nontext` directory with the name of `mars.doc`.
+Back in the terminal, run the usual commands for setting up a new Mercurial repository:
+
+~~~ {.bash}
+$ hg init
+$ hg add mars.doc
```
Comment (kynan) [inline discussion.md:85]: Later on you're using `mars.odt`

File: discussion.md [hunk 4]
```diff
@@ -3,4 +3,131 @@ layout: page
...
++++ b/mars.txt  Mon Apr 14 15:48:53 2014 -0400
+@@ -1,1 +1,2 @@
+ Cold and dry, but everything is my favorite color
++The two moons may be a problem for Wolfman
+~~~
+
+Notice how plain text files give a much more informative diff.
+You can see exactly which lines changed and what the changes were.
+
+An uninformative `hg diff` is not the only consequence of using Mercurial on binary files.
```
Comment (kynan) [inline discussion.md:121]: Here you could mention that a new copy has to be stored whenever a binary file changes and hence much more space is needed.

**Comment by douglatornell** [comment]
Thanks for the review @kynan. Those are all good points that I will fix/incorporate before I merge this PR.

I really appreciate your help in whipping this lesson into shape for the release.

**Pull request merged**